Remove DKMS module updater

This patch disables the in target installation of DKMS module updater
mechanism unneeded in buildroot.

Signed-off-by: Angelo Compagnucci <angelo.compagnucci@gmail.com>

--- a/driver/CMakeLists.txt
+++ b/driver/CMakeLists.txt
@@ -39,21 +39,3 @@ add_custom_target(install_driver
 	WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
 	VERBATIM)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/Makefile.dkms
-	RENAME Makefile
-	DESTINATION "src/sysdig-${SYSDIG_VERSION}")
-
-install(FILES
-	${CMAKE_CURRENT_BINARY_DIR}/dkms.conf
-	dynamic_params_table.c
-	event_table.c
-	flags_table.c
-	main.c
-	ppm.h
-	ppm_events.c
-	ppm_events.h
-	ppm_events_public.h
-	ppm_fillers.c
-	ppm_ringbuffer.h
-	syscall_table.c
-	DESTINATION "src/sysdig-${SYSDIG_VERSION}")
